packages/flutter_tools/lib/src/commands/devices.dart

```diff
@@ -31,7 +31,7 @@ class DevicesCommand extends FlutterCommand {
 
       for (Device device in devices) {
         String supportIndicator = device.isSupported() ? '' : '- unsupported';
-        printStatus('${device.name} (${device.id}) ${supportIndicator}');
+        printStatus('${device.name} (${device.id}) $supportIndicator');
```

Comment:
nit: `' - unsupported'`

Add a space before the dash, and remove it from the printStatus() line.